### DISCUSSION TITLE
Update documentation

### DISCUSSION THREAD

**Discussion by olarno** [CodeReviewPals/code-review-pals#64] [Ideas]
Hello, 
I think it would be good to update the documentation for new developers. 
- How to install locally. - At least
- How to start your contribution, your first problem, your first report.
- How to make a good review 

Maybe add more details about the current stack: Laravel / react with their version. 

Another question: do you have a template for PR?

Just a suggestion to be more readable.

**Comment by vassilidev** [comment]
Yes i think its a good thing to do, lets make an issue !

**Comment by geeksesi** [comment]
@olarno could you send some PR templates?
